plugins/source/oracledb/client/types.go

```diff
@@ -49,14 +49,16 @@ func SQLType(t arrow.DataType) string {
 	case *arrow.Int8Type, *arrow.Int16Type, *arrow.Int32Type, *arrow.Int64Type,
 		*arrow.Uint8Type, *arrow.Uint16Type, *arrow.Uint32Type, *arrow.Uint64Type:
 		return "NUMBER(38)"
-	case *arrow.Float16Type, *arrow.Float32Type, *arrow.Float64Type:
+	case *arrow.Float16Type, *arrow.Float32Type:
```

Comment:
This code is only used in the tests.
Previously the OracleDB types were not accurate enough and data was lost during insert. It was not discovered in the tests, due to doing string comparison